app/controllers/orga/conferences_controller.rb

```diff
@@ -1,31 +1,21 @@
 class Orga::ConferencesController < Orga::BaseController
   before_action :find_conference, only: [:show, :edit, :update, :destroy]
 
+  def import
+    Conference::Importer.import(params[:file])
+    redirect_to orga_conferences_path, notice: "Import finished! Check log for errors."
+  end
+  
   def index
     @conferences = conferences
   end
   
   def new
-    @conference = Conference.new
   end
   
   def create
-    @conference = Conference.new(conference_params.merge(season: current_season))
-    if @conference.save
-     redirect_to orga_conference_path(@conference)
-    else
-      render action: :new
-    end
   end
   
-  def update
-    if @conference.update(conference_params)
-      redirect_to orga_conference_path(@conference)
-    else
-      render action: :edit
-    end
-  end
-
   def destroy
```

Comment:
The destroy action is not routed anymore. The implementation is still here though and I think it makes sense to be able to delete a conference. Can you add it to the `routes.rb` again ?